freqtrade/plot/plotting.py

```diff
@@ -48,11 +49,21 @@ def init_plotscript(config):
         data_format=config.get('dataformat_ohlcv', 'json'),
     )
 
-    trades = load_trades(config['trade_source'],
-                         db_url=config.get('db_url'),
-                         exportfilename=config.get('exportfilename'),
-                         )
+    skip_trades = False
+    if not isfile(config.get('exportfilename')) and config['trade_source'] == 'file':
```

Comment:
#3064 should help this by having exportfilename as `pathlib.Path` right out of the configuration - so `config['exportfilename'].is_file()` can be used.

Also, if you look at `configuration.py` - this parameter is always set to the default - so it's safe to assume it'll always be there.

Reply:
merged #3064 to develop and resolved conflict here (pls verify it)

Reply:
It works in latest commit